src/ProgressBar/ProgressBar.scss

```diff
@@ -7,4 +7,80 @@
 .progress-bar { 
     background-color: $progress-bar-bg;
 }
+
+.pgn__progress-annotated {
+    width: 100%;
+    position: relative;
+    overflow: visible;
+
+    .progress {
+        overflow: visible;
+        background-color: $light-300;
+        height: $annotated-progress-height;
+        border: none;
+
+        .progress-bar {
+            background-color: $annotated-progress-bar-bg;
+            overflow: visible;
+            position: relative;
+        }
+
+        .pgn__progress-tick--white::after,
+        .pgn__progress-tick--black::after {
+            content: '';
+            position: absolute;
+            height: $annotated-progress-height;
+            width: 1px;
+            right: 0;
+        }
+
+        .pgn__progress-tick--white::after {
+            background: $light-300;
+        }
+
+        .pgn__progress-tick--black::after {
+            background: $primary-500;
+        }
+
+        @each $name, $color in $progress-colors {
+            .pgn__progress-bar--#{$name} {
+                background-color: $color;
+            }
+
+            .pgn__progress-threshold-dot--#{$name} {
+                background: $color;
+                position: absolute;
+                margin-top: -($progress-threshold-circle / 2 - $annotated-progress-height / 2);
+                margin-left: -($progress-threshold-circle / 2);
+                width: $progress-threshold-circle;
+                height: $progress-threshold-circle;
+                border-radius: $progress-threshold-circle / 2;
+                z-index: 1;
+            }
+        }
+    }
+
+    .progress::before,
+    .progress::after {
+        position: absolute;
+        content: '';
+        height: $annotated-progress-height;
+        width: 1px;
+        background: $primary-500;
+    }
+
+    .progress::after {
+        right: 0;
+    }
+
+    .pgn__progress-info {
+        position: relative;
+    }
```

Comment:
It looks like these div's seem to be expanding beyond the container and causing horizontal scrolling on the page.

<img width="1453" alt="image" src="https://user-images.githubusercontent.com/2828721/158994108-16955eec-4123-4e1c-9b9c-4a1fb1afdcc2.png">

Reply:
`display: inline-block` fixed the issue